Source/WebCore/css/calc/CSSCalcTree.h

```diff
@@ -704,6 +707,24 @@ struct Sign {
     bool operator==(const Sign&) const = default;
 };
 
+
+struct Anchor {
+    WTF_MAKE_TZONE_ALLOCATED_INLINE(Anchor);
+public:
```

Comment:
Isn’t there a version of the macro for `struct` that leaves things `public`? I guess maybe not.

Reply:
I don't see one, feels like there should be.